### PR TITLE
[DPS-2773] Improvements to the Building webhook apps article

### PR DESCRIPTION
**Pull request by oliwiapolec**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/DPS-2773)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2773)

## 📓 Description

This PR includes updates to the [Building webhook apps](https://developers.livechat.com/docs/getting-started/livechat-apps/webhook-apps/) article, in scope:
- changed screenshots;
- improved copy;

### Content
- Proofreading/content fixes

### CODE REVIEW

File: content/getting-started/livechat-apps/webhook-apps/index.mdx [hunk 1]
```diff
@@ -9,102 +9,106 @@ desc: "Learn how to build your own integration using LiveChat webhooks."
...
 
 - [Configuration API v3](/management/configuration-api/#webhooks)
 - [Configuration API v2](/management/configuration-api/v2.0/#create-a-new-webhook) (deprecated)
 
 ### Seting up webhooks for development
 
-For development purposes we recommend setting your webhooks from within LiveChat Agent App.
+One of the possible development options is to set up your webhooks from within the LiveChat Agent App. However, please note that this configuration uses [Configuration API v2 webhooks](/management/configuration-api/v2.0/#create-a-new-webhook), which are now deprecated.
+
+We highly recommend using the [Configuration API v3 webhooks](/management/configuration-api/#webhooks). To view a tutorial on setting these up, follow [this link](/getting-started/app-guides/chat-webhooks/#tutorial).
```
Comment (ale-kacperczyk) [inline content/getting-started/livechat-apps/webhook-apps/index.mdx:80]:
```suggestion
We highly recommend using the [Configuration API v3 webhooks](/management/configuration-api/#webhooks).

Comment (ale-kacperczyk) [reply content/getting-started/livechat-apps/webhook-apps/index.mdx:80]: It's better not to link that at the moment. v3.3 introduces important changes that we'll need to include in this docs

File: content/getting-started/livechat-apps/webhook-apps/index.mdx [hunk 2]
```diff
@@ -9,102 +9,106 @@ desc: "Learn how to build your own integration using LiveChat webhooks."
...
+To configure webhooks v2 within the LiveChat Agent App, go to **Settings > Integrations > [Webhooks](https://my.livechatinc.com/settings/webhooks)** and set up your webhook URLs.
 
 ![LiveChat Webhooks](./images/livechat-platform-webhooks-settings.png)
 
 Each webhook consists of the following properties:
 
 - **Event** – determines when the webhook is sent to your web server.
-- **Data type** – includes additional information in the webhook.
+- **Data type** – allows you to select which information will be included in the webhook.
 - **Target URL** – address of your web server the webhook will be sent to.
```
Comment (ale-kacperczyk) [inline content/getting-started/livechat-apps/webhook-apps/index.mdx:90]:
```suggestion
- **Target URL** – the address of your web server the webhook will be sent to.
```

File: content/getting-started/livechat-apps/webhook-apps/index.mdx [hunk 3]
```diff
@@ -9,102 +9,106 @@ desc: "Learn how to build your own integration using LiveChat webhooks."
...
+**Note: This reference is deprecated**. Versions 3.x of Platform APIs introduce new webhook interfaces. We highly recommend migrating out of the deprecated system. The new webhooks service has much more functionality and provides you with a higher control over the data. Please note that the new Platform APIs also provide a **RTM (websocket) transport**.
 
 </Warning>
 
-LiveChat can send notifications when some particular action is performed. Such a notification is called a **webhook** – it’s just a simple HTTP request that LiveChat sends to your server when a particular event occurs. Check the basic auth example below.
+LiveChat can send notifications when some specific action is performed. Such a notification is called a **webhook** – it’s a simple HTTP request that LiveChat sends to your server when a particular event occurs. You can find a basic auth example below.
```
Comment (ale-kacperczyk) [inline content/getting-started/livechat-apps/webhook-apps/index.mdx:107]:
```suggestion
LiveChat can send notifications when some specific actions are performed. Such a notification is called a **webhook** – it’s a simple HTTP request that LiveChat sends to your server when a particular event occurs. You can find a basic auth example below.
```

File: content/getting-started/livechat-apps/webhook-apps/index.mdx [hunk 4]
```diff
@@ -154,7 +156,7 @@ Each webhook is a HTTP POST request made to the URL that you provide in the web
 Each webhook contains the following properties:
 
 - `event_type` – tells you the event that triggered the webhook. Possible values: `chat_started`, `chat_ended`, `visitor_queued`.
-- `token` and `license_id` – your authentication credentials that let you call LiveChat’s Configuration API v2 methods. You won’t need to use them unless you want to make a call to LiveChat’s API right when you receive a webhook. In that case, you just need to pass these `token` and `license_id` credentials in your API call.
+- `token` and `license_id` – your authentication credentials that let you call LiveChat’s Configuration API v2 methods. You won’t need to use them unless you want to make a call to LiveChat’s API right when you receive a webhook. In that case, you just need to pass the `token` and `license_id` credentials in your API call.
```
Comment (ale-kacperczyk) [inline content/getting-started/livechat-apps/webhook-apps/index.mdx:159]:
```suggestion
- `token` and `license_id` – your authentication credentials that let you call LiveChat Configuration API v2 methods. You won’t need to use them unless you want to make a call to the LiveChat API right when you receive a webhook. In that case, you just need to pass the `token` and `license_id` credentials in your API call.
```

File: content/getting-started/livechat-apps/webhook-apps/index.mdx [hunk 5]
```diff
@@ -176,7 +178,7 @@ That would prevent situations in which the process takes longer than our webhook
 
 Note: LiveChat webhooks are sent with `Content-Type: application/json` header, so please make sure that your service can handle such requests.
 
-Note: `chat_changed` event type is sent only when the tag list for a particular chat was changed, no matter if the chat is pending or not.
+Note: `chat_changed` event type is sent only when the tag list for a particular chat was changed, no matter if the chat is currently ongoing or not.
```
Comment (ale-kacperczyk) [inline content/getting-started/livechat-apps/webhook-apps/index.mdx:181]: I wonder what the author originally meant by "pending"? 🤔 Queued, maybe?

File: content/getting-started/livechat-apps/webhook-apps/index.mdx [hunk 6]
```diff
@@ -197,9 +199,9 @@ Note: `chat_changed` event type is sent only when the tag list for a particular
 
 </CodeResponse>
 
-In some cases, you may want to get some additional information when the particular event occurs.
+In some cases, you may want to get some additional information when a particular event occurs.
 
-For example, when the chat starts, you may want to know the exact chat start time along with the visitor's name and e-mail provided in the pre-chat survey. To get this information, you can add some **data types** that will be sent to your webserver along with each webhook.
+For example, when the chat starts, you may want to know the exact chat start time along with the visitor's name and e-mail provided in the pre-chat survey. To get this information, you can add additional **data types** that will be sent to your web server along with each webhook.
```
Comment (ale-kacperczyk) [inline content/getting-started/livechat-apps/webhook-apps/index.mdx:204]:
```suggestion
For example, when the chat starts, you may want to know the exact chat start time along with the visitor's name and email provided in the pre-chat survey. To get this information, you can add additional **data types** that will be sent to your web server along with each webhook.
```

**Review by ale-kacperczyk — COMMENTED** [review]
Great job :)

File: content/getting-started/livechat-apps/webhook-apps/index.mdx (hunk 5 above)
Comment (oliwiapolec) [reply content/getting-started/livechat-apps/webhook-apps/index.mdx:181]: As far as I remember the chat_changed webhook triggers anytime a tag is changed, meaning during or after the chat - so the ongoing (in my mind) meant that the chat is currently live, not already archived :D

**Review by jakubsikora — APPROVED** [review]

File: content/getting-started/livechat-apps/webhook-apps/index.mdx [hunk 7]
```diff
@@ -9,102 +9,121 @@ desc: "Learn how to build your own integration using LiveChat webhooks."
...
 
-- ask users to connect their accounts with your OAuth2.0 provider,
-- display configuration options,
-- inform users about additional installation steps (if there are any).
+- Prompt users to connect their accounts with your OAuth2.1 provider.
+- Display configuration options.
+- Inform users about additional installation steps (if there are any).
 
-If you need to access LiveChat user data, we recommend using [Sign in with LiveChat SDK](/getting-started/authorization/sign-in-with-livechat/).
+If you need to access LiveChat user data, we recommend using [Sign in with LiveChat authorization flow](/getting-started/authorization/sign-in-with-livechat/). For more information on possible widget placements, visit the [app locations](/extending-agent-app/#app-locations) page.
```
Comment (ale-kacperczyk) [inline content/getting-started/livechat-apps/webhook-apps/index.mdx:84]:
```suggestion
If you need to access LiveChat user data, we recommend using the [Sign in with LiveChat flow](/getting-started/authorization/sign-in-with-livechat/). For more information on possible widget placements, visit the [app locations](/extending-agent-app/#app-locations) page.
```

File: content/getting-started/livechat-apps/webhook-apps/index.mdx [hunk 8]
```diff
@@ -9,102 +9,121 @@ desc: "Learn how to build your own integration using LiveChat webhooks."
...
 
 ### Sample settings page scenario
 
-For instance if you're building an integration that binds LiveChat data with an external service, you should create a settings page that:
+If you're building an integration that binds LiveChat data with an external service, you should create a settings page that:
 
-1. Imports [Sign in with LiveChat SDK](/getting-started/authorization/sign-in-with-livechat/).
+1. Makes use of the [Sign in with LiveChat flow](/getting-started/authorization/sign-in-with-livechat/).
 2. Imports your sign in form or other authorization flow (e.g. "Connect with..." button).
```
Comment (ale-kacperczyk) [inline content/getting-started/livechat-apps/webhook-apps/index.mdx:91]: + link to https://developers.livechat.com/docs/authorization/authorizing-api-calls/#authorization-code-grant

**Review by ale-kacperczyk — APPROVED** [review]
Feel free to merge it tomorrow :)

**Pull request merged**